test/integration/targets/copy/tasks/main.yml

```diff
@@ -169,8 +169,9 @@
 # Test copy with mode=preserve
 - name: Set file perms to an odd value
   file:
-    path: '{{ output_file }}'
+    path: '{{ role_path }}/files/foo.txt'
```

Comment:
Updated: another `foo.txt` file is created on the controller side, in the temporary directory (https://github.com/ansible/ansible/pull/25672/commits/e0a71a51276a528c2c013c6e4cde8d36d844ec6b#diff-6ce41036c58266d9f14a165103092878).